pkg/test/toolchain/toolchain.go

```diff
@@ -3,23 +3,9 @@ package toolchain
 import (
 	"fmt"
 	"time"
-
-	"github.com/codeready-toolchain/toolchain-operator/pkg/apis/toolchain/v1alpha1"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// GenerateName return the given name with a suffix based on the current time (UnixNano)
 func GenerateName(prefix string) string {
```

Comment:
This file now contains only this function which is used only in one test file - `tektoninstallation_controller_test.go` - I think that we could move it there and remove this file, WDYT?

Reply:
ah sorry, I missed that comment when merging the PR. Yeah, we could actually move it to the test file, indeed. Let's address that in a subsequent PR

Reply:
@MatousJobanek addressed in https://github.com/codeready-toolchain/toolchain-operator/pull/20 ;)